account_banking_sepa_direct_debit/views/account_banking_mandate_view.xml

```diff
@@ -12,6 +12,15 @@
         <field name="model">account.banking.mandate</field>
         <field name="inherit_id" ref="account_banking_mandate.view_mandate_form" />
         <field name="arch" type="xml">
+            <button name="validate" position="before">
+                <button
+                    name="print_report"
+                    type="object"
+                    string="Print"
+                    class="btn-default"
+                    states="draft,valid"
```

Comment:
Once valid, it should be shown IMO, or at least not as highlighted.

Reply:
ok, I agree. I removed states=""

Reply:
Agh, I made a typo, I wanted to say `shouldn't be shown`, but well if you put it without states, but not highlighted, it's OK.

Reply:
Please remove then `class="btn-default"`

Reply:
btn-default is NOT highlighted (it's highligthed when you have btn-primary).

Reply:
Ah, OK, sorry for the confusion.